clouds/cli/README.md

```diff
@@ -0,0 +1,47 @@
+# Cloud nuke CLI tools
+* Run `make cloud-cli` to run with `cloud-cli` (requires root privileges).
```

Comment:
why do we require root?

Reply:
to run `sudo mv dist/cli /usr/local/bin/cloud-cli`
must be a better way to do this

Reply:
use pipx, we do not use sudo to install cli tools.

Reply:
done